source/scripts/fetch-stats.js

```diff
@@ -0,0 +1,266 @@
+// This script uses the Matomo API which requires an access token
+// Once you have your access token you can put it in a `.env` file at the root
+// of the project to enable it during development. For instance:
+//
+// MATOMO_TOKEN=f4336c82cb1e494752d06e610614eab12b65f1d1
+//
+// Matomo API documentation:
+// https://developer.matomo.org/api-reference/reporting-api
+
+require('dotenv').config()
+require('isomorphic-fetch')
+const querystring = require('querystring')
+const { createDataDir, writeInDataDir } = require('./utils.js')
+const R = require('ramda')
+
+const apiURL = params => {
+	const query = querystring.stringify({
+		period: 'month',
+		date: 'last1',
+		method: 'API.get',
+		format: 'JSON',
+		module: 'API',
+		idSite: 39,
+		language: 'fr',
+		apiAction: 'get',
+		token_auth: process.env.MATOMO_TOKEN,
+		...params
+	})
+	return `https://stats.data.gouv.fr/index.php?${query}`
+}
+
+async function main() {
+	createDataDir()
+	const stats = {
+		simulators: await fetchSimulatorsMonth(),
+		monthly_visits: await fetchMonthlyVisits(),
+		daily_visits: await fetchDailyVisits(),
+		status_chosen: await fetchStatusChosen(),
+		feedback: await fetchFeedback()
+	}
+	writeInDataDir('stats.json', stats)
+}
+function FirstDayXMonthAgo(dt, X = 0) {
```

Comment:
```suggestion
function firstDayXMonthAgo(dt, X = 0) {
```

Ah oui pareil on nomme les fonctions avec une première lettre en minuscules et les composants React avec une majuscule. (<kbd>F2</kbd> Pour renommer toutes les occurrences dans VSCode)